src/components/RegistrationList.tsx

```diff
@@ -47,66 +78,131 @@ export const RegistrationListItemContent = ({ registration, ticketView = false }
-                  </MuiLink>
-                ) : (
-                  contributor.identity.name
-                )}
+
+          {ticketView &&
+            (registration.status === RegistrationStatus.Draft || registration.status === RegistrationStatus.New) && (
+              <Typography
+                sx={{
+                  p: '0.1rem 0.75rem',
+                  bgcolor: 'primary.light',
+                  color: 'primary.contrastText',
+                }}>
+                {t('registration.public_page.metadata_not_published')}
               </Typography>
-              <ContributorIndicators contributor={contributor} ticketView={ticketView} />
-            </Box>
-          ))}
-          {countRestContributors > 0 && (
-            <Typography variant="body2">({t('common.x_others', { count: countRestContributors })})</Typography>
+            )}
+        </Box>
+        <Typography gutterBottom sx={{ fontSize: '1rem', fontWeight: '600', wordWrap: 'break-word' }}>
+          {ticketView ? (
+            getTitleString(entityDescription?.mainTitle)
+          ) : (
+            <MuiLink component={Link} to={getRegistrationLandingPagePath(identifier)}>
+              {getTitleString(entityDescription?.mainTitle)}
+            </MuiLink>
           )}
+        </Typography>
+        <Box
+          sx={{
+            display: 'flex',
+            flexWrap: 'wrap',
+            columnGap: '1rem',
+            whiteSpace: 'nowrap',
+          }}>
+          <Box sx={{ display: 'flex', alignItems: 'center', columnGap: '1rem', flexWrap: 'wrap' }}>
+            {focusedContributors.map((contributor, index) => (
+              <Box key={index} sx={{ display: 'flex', alignItems: 'center' }}>
+                <Typography variant="body2">
+                  {contributor.identity.id && !ticketView ? (
+                    <MuiLink component={Link} to={getResearchProfilePath(contributor.identity.id)}>
+                      {contributor.identity.name}
+                    </MuiLink>
+                  ) : (
+                    contributor.identity.name
+                  )}
+                </Typography>
+                <ContributorIndicators contributor={contributor} ticketView={ticketView} />
+              </Box>
+            ))}
+            {countRestContributors > 0 && (
+              <Typography variant="body2">({t('common.x_others', { count: countRestContributors })})</Typography>
+            )}
+          </Box>
         </Box>
-      </Box>
 
-      <TruncatableTypography sx={{ mt: '0.5rem', maxWidth: '60rem' }}>
-        {entityDescription?.abstract}
-      </TruncatableTypography>
-    </ListItemText>
+        <TruncatableTypography sx={{ mt: '0.5rem', maxWidth: '60rem' }}>
+          {entityDescription?.abstract}
+        </TruncatableTypography>
+      </ListItemText>
+
+      {canEditRegistration && (
+        <>
+          <Box sx={{ display: 'flex', alignItems: 'start', justifySelf: 'end', gap: '0.5rem' }}>
```

Comment:
burde ikke trenge justifySelf her?
```suggestion
          <Box sx={{ display: 'flex', alignItems: 'start', gap: '0.5rem' }}>
```